packages/devtools_app/lib/src/screens/debugger/codeview.dart

```diff
@@ -418,6 +418,8 @@ class _CodeViewState extends State<CodeView>
     );
   }
 
+  InlineSpan span(InlineSpan span) => span;
```

Comment:
This shouldn't be necessary since it's just converting the inline span into an inline span